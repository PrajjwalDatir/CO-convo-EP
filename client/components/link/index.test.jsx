import React from 'react';
import { mount } from 'enzyme';
import { StaticRouter } from 'react-router-dom';
import Link from './index';

describe('<Link />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(
      <StaticRouter>
        <Link to="/" />
      </StaticRouter>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
