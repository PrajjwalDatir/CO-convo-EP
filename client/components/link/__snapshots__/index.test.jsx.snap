// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> should match snapshot 1`] = `
<StaticRouter>
  <Router
    history={
      Object {
        "action": "POP",
        "block": [Function],
        "createHref": [Function],
        "go": [Function],
        "goBack": [Function],
        "goForward": [Function],
        "listen": [Function],
        "location": Object {
          "hash": "",
          "pathname": "/",
          "search": "",
          "state": undefined,
        },
        "push": [Function],
        "replace": [Function],
      }
    }
    staticContext={Object {}}
  >
    <Link
      to="/"
    >
      <Link
        className="text-blue-600 underline hover:text-blue-700"
        to="/"
      >
        <LinkAnchor
          className="text-blue-600 underline hover:text-blue-700"
          href="/"
          navigate={[Function]}
        >
          <a
            className="text-blue-600 underline hover:text-blue-700"
            href="/"
            onClick={[Function]}
          />
        </LinkAnchor>
      </Link>
    </Link>
  </Router>
</StaticRouter>
`;
